app/ui.R

```diff
@@ -81,6 +81,11 @@ ui <- fluidPage(
             sliderInput('indirect_contacts', 
                         'Indirect exposure: how many people in total do your household members come into close contact with in a week? (Do not include contact between household members in this count.)', 
                         min = 0, max = 100, value = 0, step =1)),
+          hr(),
+          checkboxInput("hand", div("I perform hand hygiene according to ", 
+                                    tags$a("CDC guidance", href = urls$cdc_hand_hygiene))),
+          checkboxInput("ppe", div("I wear personal protection equipment consistent with ",
+                                   tags$a("CDC guidelines", href = urls$cdc_ppe))),
```

Comment:
Emma, could you also update the URL for CDC PPE guideline to be the most recent? Thanks!

Reply:
Is this the URL you are thinking of: https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/cloth-face-cover-guidance.html

Reply:
@pendlrobinson This is great, thanks!